task2-bash/backup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -ex
+
+src_folder="../task1-simple-program"
+tmp_folder="tmp/guesanumber"
+
+
+## Copying files to tmp folder
+mkdir -p $tmp_folder
+cp $src_folder/*.{c,h} $tmp_folder
```

Comment:
Paths in quotes is a good practice

Reply:
Fixed https://github.com/Kernel-GL-HRK/gl_kernel_BaseCamp_2022_II/pull/33/commits/7db8264394124e8030fcdbdcfa961a1690d659c1